src/iptools-jquery-accordion.js

```diff
@@ -92,7 +92,9 @@
     init: function() {
 
       this.$panels = this.$element.children(selectorFromClass(this.settings.panelClass));
-      var $activePanels = this.$panels.filter(selectorFromClass(this.settings.panelActiveClass));
+      var $activePanels = this.$panels
+        .filter(selectorFromClass(this.settings.panelActiveClass))
+        .add(this.$panels.filter('[data-accordion-expanded="true"]'));
```

Comment:
Agreed Re consistency. What about `data-accordion-panel-activated` to have better semantics in terms of readability?